### ISSUE TITLE
Web server after update only text mode (Ticket-507)

### ISSUE THREAD

**Issue by pilepalle** [nextcloud/nextcloudpi#511]
Hy Guys,

thank you for fixing Problem 507 (nc-datadir not updating apache logs # 507).
After the fix, my web presentation will not work properly anymore. The web server only displays everything in text. Does anyone have an idea what that can be and how can I fix it?

![web](https://user-images.githubusercontent.com/38583562/40304851-83412eb4-5cf8-11e8-8530-4928c0e8d16d.png)

Thanks

**Comment by pilepalle** [comment]
I just noticed that when I open the Nextcloud via the local IP address the web view does not come to its 
full extent (icons and co). The view via DNS Web externally displays only the text version.

![image](https://user-images.githubusercontent.com/38583562/40329593-4f240dac-5d4a-11e8-980f-a1f1cb094f55.png)

**Comment by nachoparker** [comment]
either your pi is really busy doing something right now, or you have hardware issues (bad powersupply/SD card)

you have tried rebooting right?

**Comment by chiyi4488** [comment]
Me too. I have the same question.

**Comment by chiyi4488** [comment]
I think this is not a "hardware" problem, it SSL.
because if I add a new domain and connection to my ncp.It can normal display.

**Comment by chiyi4488** [comment]
but if I update NC to 14.0.0 (13.0.6 to 14.0.0),I can normal display.
and reboot........ text mode :D~~~
why?

**Comment by TomTurnschuh** [comment]
@chiyi4488: Could you please try with a different browser?
I once had this issue after a hardware change. With a different browser it worked – and later also with the original browser.

**Comment by chiyi4488** [comment]
no, I can't. I change my browser to edge and IE. It same.

**Comment by chiyi4488** [comment]
Wait, I used the update software built into the nextcloud to update (previously applied to ncp), the system is back to normal, no longer "text mode", very strange. I think this may be caused by some kind of error that nc-restore and nc-nextcloud-update have not found yet.

I have recovered nc before. (nc-restore)

**Comment by TomTurnschuh** [comment]
>  I think this may be caused by some kind of error that nc-restore and nc-nextcloud-update have not found yet.

Well, maybe there is indeed an unresolved issue with nc-restore. However, I rather think it is a matter of browser cache. @chiyi4488, have you connected to your cloud with the mentioned browsers already before the restore, or only after? If you also did before, then deleting the browser cache could be a reasonable solution.

**Comment by k3tan25** [comment]
I'm also getting 'text mode'. Not sure why. Every browser. Every device. And no, this has nothing to do with the power supply.

**Comment by chiyi4488** [comment]
I am guessing that there may be a problem with the way nc-restore is restored. @k3tan25 Did you show this after restore?

**Comment by k3tan25** [comment]
No, I haven't used the restore function at all. Freshly installed. Ran some updates for ncp and nc. Not sure what's triggered it.

**Comment by chiyi4488** [comment]
According to my guess, if ncp replaces the nc file, it is very likely that something like this will happen. But we still don't know why this is happening.

@nachoparker Any ideas?

**Comment by chiyi4488** [comment]
Maybe we should compare the ncp file with the nc white paper, but I didn't subscribe. I can't view it.

**Comment by chiyi4488** [comment]
https://github.com/DecaTec/Nextcloud-Backup-Restore
https://portal.nextcloud.com/article/13/backup-restore-30.html

**Comment by nachoparker** [comment]
this has nothing to do with backup and restore
